packages/backend-for-frontend/src/services/clientService.ts

```diff
@@ -353,6 +358,7 @@ async function enhanceClient(
       name: consumer.name,
     },
     purposes,
+    admin,
```

Comment:
```suggestion
    admin: admin
      ? {
          id: admin.id,
          name: admin.name,
        }
      : undefined,
```
We don't have to return the whole tenant